js/views/app_view.js

```diff
@@ -16,6 +16,9 @@
             'change-theme': 'applyTheme'
         },
         applyTheme: function() {
+            var hideMenuBar = storage.get('hide-menu-bar', false);
+            window.setAutoHideMenuBar(hideMenuBar);
+            window.setMenuBarVisibility(!hideMenuBar);
```

Comment:
Oh I see, applyTheme is now more like applyAllTheThings ?

Reply:
I found menu bar and theme close enough for it to be in the same function.
Do you want me to move it?

Reply:
I think a separate event would be good. Something like 'change-menu' instead of 'change-theme.'